components/FooterCard.vue

```diff
@@ -90,3 +90,37 @@
   }
 }
 </style>
+
+<script>
+import sharedState from './sharedState'
+
+const defaultUrl = 'https://www.votdiaspora.md'
+
+export default {
+  data() {
+    return sharedState
+  },
+  computed: {
+    votdiasporaUrl() {
+      const { lat, long } = this.coordinates
+      if (lat && long) {
+        return `${defaultUrl}?lat=${lat}&long=${long}`
+      }
+      return defaultUrl
+    },
```

Comment:
Dar sigur se face update la `votdiasporaUrl` cand in componenta `PollingStationSearch` se seteaza `sharedState.coordinates`? 🤔

Reply:
Pentru a verifica cel mai simplu e în [preview](https://vot-diaspora-client-git-fork-vtzk-gh-62-bet-8d1cbc-code4moldova.vercel.app) de făcut search la o oarecare locație, după care apeși pe `mailto` se va deschide clientul de mail cu link-ul în body. Pentru mine funcționează. 
De asemeni și cînd apeși pe twitter. Facebook nu am verificat (nu folosesc FB).

Reply:
Prima dată scriu în app `vue.js`, nu cunosc metode (simple/rapide) de state sharing între componente. 
Aparent acest shared state e transformat automat în Observable.

Reply:
Am si uitat ca avem preivew :)
Ok, vad ca lucreaza, dar nu inteleg de ce. 😄